itou/www/job_seekers_views/views.py

```diff
@@ -363,7 +375,7 @@ def post(self, request, *args, **kwargs):
             # TODO(ewen): check_job_seeker_info doesn't use the session yet,
             # so we delete the session here.
             self.job_seeker_session.delete()
-            return self.redirect_to_check_infos(self.job_seeker.public_id)
+            return HttpResponseRedirect(self.get_exit_url(self.job_seeker.public_id))
```

Comment:
En vrai, on est dans `CheckNIRForJobSeekerView` (je t'ai déjà donné mon avis sur cette vue :stuck_out_tongue_winking_eye: )

```suggestion
            return HttpResponseRedirect(reverse("job_seekers_views:check_job_seeker_info", kwargs={"company_pk": self.company.pk, "job_seeker_public_id": job_seeker_public_id}))
```

comme dans le `get` juste au dessus me semble plus clair ;)